executors/sshexec/volume.go

```diff
@@ -155,6 +158,33 @@ func (s *SshExecutor) VolumeDestroyCheck(host, volume string) error {
 	return nil
 }
 
+func (s *SshExecutor) createVolumeOptionsCommand(volume *executors.VolumeRequest) []string {
+
+	godbc.Require(len(volume.Options) > 0)
+
+	commands := []string{}
+	var cmd string
+
+	// Go through all the Options and create volume set command
+	for _, volOption := range volume.Options {
+		if volOption != "" {
+			switch volOption {
+			case "encryption":
+				cmd = fmt.Sprintf("gluster --mode=script volume set %v server.ssl on", volume.Name)
```

Comment:
I am not 100% comfortable with adding the wrapping heketi-specific options through the same command-line option as the pass-through options. Isn't it dangerous? What if gluster adds an "encryption" option that means an entirely different thing?

I.e. if we add heketi-specific wrapping-options, should we enable them either with a specific cmdline option like "--encryption" or else add two cmline parameters, one (e.g. --options) for the wrapping options and one (e.g. --gluster-options) for the pass-through options? I know it makes things more complicated, but seems cleaner.

This is not a veto (yet), still thinking about what would be the cleanest approach, like to be convinced...

@jarrpa @raghavendra-talur ... what do you think?